libraries/ui-shared/Button/index.jsx

```diff
@@ -8,6 +8,7 @@ const buttonTypes = [
   'regular',
   'primary',
   'secondary',
+  'checkout',
```

Comment:
Generic components should not have contextual options inside.

Reply:
What would you suggest instead for this case?